frontend/src/components/reports/study/NonconformityNotification.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { Grid, Column, Section, Button, Form, FormLabel } from "@carbon/react";
+import { FormattedMessage, useIntl } from "react-intl";
+import CustomDatePicker from "../../common/CustomDatePicker";
+import config from "../../../config.json";
+
+const NonconformityNotification = () => {
+  const [accessionNumber, setAccessionNumber] = useState('');
+  const [selectedSite, setSelectedSite] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [notificationMessage, setNotificationMessage] = useState('');
+  const [error, setError] = useState('');
+  
+  
+  
+  const siteOptions = [
+    { label: '', value: '' },
+    { label: 'Site 1', value: 'site1' },
+    { label: 'Site 2', value: 'site2' },
+  ];
+
+  const handleAccessionNumberChange = (event) => {
+    setAccessionNumber(event.target.value);
+  };
+
+  const handleSiteChange = (event) => {
+    setSelectedSite(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (!accessionNumber) {
+      setError('Accession number is required.');
+      return;
+    }
+    setLoading(true);
```

Comment:
for submititng , just dynalically build a url with params like below and openit in a new tab ie 
/ReportPrint?report=retroCInonConformityNotification&type=patient&accessionDirect=<accesionNumber>&selectList.selection=<SiteId>

Reply:
see [example ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/reports/routine/ReferredOut.js#L90-L105)of how its done